.github/workflows/main.yml

```diff
@@ -13,3 +13,10 @@ jobs:
       env:
         API_TOKEN: ${{ secrets.API_TOKEN }}
         PROJECT_UUID: ${{ secrets.PROJECT_UUID }}
+    - uses: meeDamian/github-release@2.0
+      if: contains(github.ref, 'tags')
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        files: dist/*demo*
```

Comment:
@erauer For publishing as GitHub Artifacts, should we not include the other build artifacts as well ?  Those are firmware images targeting PLT-supported devboards, and can be used for example test scripts.

Reply:
@IvoBCD I think it make those changes in a separate update and create test suites per dev board and upload the artifacts as zip files. 

My thought was that we should have an out of the box experience that runs the customer through the process for the demo board. Long term, the idea of giving them an option  of choosing  to do that process with the demo board or a supported dev kit makes sense.